.travis.yml

```diff
@@ -2,10 +2,10 @@ language: java
 jdk: oraclejdk8
 env:
 - MAVEN_SKIP_RC=true MAVEN_OPTS="-Xmx512m -XX:MaxPermSize=512m -Duser.language=en -Duser.country=US" POST_COVERAGE=true
-- MAVEN_SKIP_RC=true MAVEN_OPTS="-Xmx512m -XX:MaxPermSize=512m -Duser.language=de -Duser.country=DE" POST_COVERAGE=false
-- MAVEN_SKIP_RC=true MAVEN_OPTS="-Xmx512m -XX:MaxPermSize=512m -Duser.language=es -Duser.country=ES" POST_COVERAGE=false
-- MAVEN_SKIP_RC=true MAVEN_OPTS="-Xmx512m -XX:MaxPermSize=512m -Duser.language=fr -Duser.country=FR" POST_COVERAGE=false
-- MAVEN_SKIP_RC=true MAVEN_OPTS="-Xmx512m -XX:MaxPermSize=512m -Duser.language=hi -Duser.country=IN" POST_COVERAGE=false
+- MAVEN_SKIP_RC=true MAVEN_OPTS="-Xmx512m -XX:MaxPermSize=512m -Duser.language=de -Duser.country=DE" POST_COVERAGE=false SKIP_DEPENDENCY_CHECK=true
```

Comment:
While you are at it, should you consider changing `-XX:MaxPermSize=512m` to `-XX:MaxMetaspaceSize=512m`?

Reply:
good idea, but that's a separate PR :)